src/paas_charm/_gunicorn/charm.py

```diff
@@ -23,6 +25,58 @@ def _workload_config(self) -> WorkloadConfig:
             framework_name=self._framework_name, unit_name=self.unit.name
         )
 
+    def check_gevent_package(self) -> bool:
+        """Check that gevent is installed.
+
+        Returns:
+            True if gevent is installed.
+        """
+        pip_list_command = self._container.exec(["pip", "list"])
+        list_output = pip_list_command.wait_output()[0]
+        return "gevent" in list_output
```

Comment:
I think this is a bit brittle as, for example, a package could be called gevents and this would pass but gevent is not installed

Reply:
Thank you for pointing this out. Fixed it. Now I check it by importing the package and calling `__version__` on it.